web/src/admin/sources/SourceWizard.ts

```diff
@@ -7,73 +7,35 @@ import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
 import { AKElement } from "@goauthentik/elements/Base";
 import "@goauthentik/elements/forms/ProxyForm";
 import "@goauthentik/elements/wizard/FormWizardPage";
+import {
+    TypeCreateWizardPage,
+    TypeCreateWizardPageLayouts,
+} from "@goauthentik/elements/wizard/TypeCreateWizardPage";
 import "@goauthentik/elements/wizard/Wizard";
-import { WizardPage } from "@goauthentik/elements/wizard/WizardPage";
 
 import { msg, str } from "@lit/localize";
 import { customElement } from "@lit/reactive-element/decorators/custom-element.js";
 import { CSSResult, TemplateResult, html } from "lit";
 import { property } from "lit/decorators.js";
 
 import PFButton from "@patternfly/patternfly/components/Button/button.css";
-import PFForm from "@patternfly/patternfly/components/Form/form.css";
-import PFRadio from "@patternfly/patternfly/components/Radio/radio.css";
 import PFBase from "@patternfly/patternfly/patternfly-base.css";
 
 import { SourcesApi, TypeCreate } from "@goauthentik/api";
 
 @customElement("ak-source-wizard-initial")
-export class InitialSourceWizardPage extends WizardPage {
```

Comment:
It's always nice to see a change that just makes a whole bunch of code disappear.  :-)